components/example-notebook-servers/base/Dockerfile

```diff
@@ -99,7 +108,7 @@ ENV LANGUAGE en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
 
 # s6 - copy scripts
-COPY --chown=${NB_USER}:users --chmod=755 s6/ /etc
+COPY --chown=${NB_USER}:${NB_GID} --chmod=755 s6/ /etc
```

Comment:
actually, no, correctly set group is very important, because on openshift uid will be random, so the applicable perms will be through "group" and "other"

Reply:
It is definitely necessary to change the group owner, otherwise one of the whole ideas of this PR (to allow arbitrary UIDs) would not work.

Or are you saying that COPY will automatically make the files owned by the correct group?

Either way, its nice to be explicit about it, because those files do need to be owned by `jovyan:0`.